app/views/categories/show.html.erb

```diff
@@ -54,4 +53,10 @@
   <%= link_to "Edit Category", edit_category_path(@category), class: 'btn btn-default' %>
   <%= category.make_deactivate_btn %>
   <%= link_to "View All Categories", categories_path, class: 'btn btn-default' %>
+  <section id="models">
```

Comment:
Making this a section leaves a pretty big space above the table heading (see image in general comment). Also, let's try to center our cell contents and widen the first column.